docs/user-guides/client.md

```diff
@@ -275,7 +275,7 @@ Then call `rank`, you can feed it with multiple Documents as a list:
 ```python
 from clip_client import Client
 
-c = Client(server='grpcs://demo-cas.jina.ai:2096')
+c = Client(server='grpc://0.0.0.0:23456')
```

Comment:
why not `grpcs://api.clip.jina.ai:2096` here?

Reply:
I don't want to add <token> everywhere